components/teams-service/server/v2/server_test.go

```diff
@@ -63,6 +64,13 @@ func runAllServerTests(
 	t.Helper()
 	defer close()
 
+	t.Run("GetVersion", func(t *testing.T) {
+		resp, err := cl.GetVersion(context.Background(), &version.VersionInfoRequest{})
+		require.NoError(t, err)
+		// Version is injected via linker flags that we don't set during tests
```

Comment:
[nit] we could set it though, couldn't we? It's a package-level string variable IIRC.

Reply:
I'm not sure how do set it. Maybe we could pair on it real quick?

Reply:
We can, however, should you want to follow the breadcrumbs:

- We inject the version into the linker args here https://github.com/chef/automate/blob/31442fe17fe52e537809ca6bf9ea21eb729f636a/components/teams-service/Makefile#L9
- That includes the path of the string constant that's set, i.e. `-X ${LIBRARY_PATH}/version.Version=${BUILD_TIME}`, which, [using `LIBRARY_PATH` from `Makefile.common_go`](https://github.com/chef/automate/blob/31442fe17fe52e537809ca6bf9ea21eb729f636a/Makefile.common_go#L9) resolves to `github.com/chef/automate/lib//version.Version`.
- That defined here: https://github.com/chef/automate/blob/31442fe17fe52e537809ca6bf9ea21eb729f636a/lib/version/version.go#L10

With that, I'd try to set it to something by importing `github.com/chef/automate/lib/version`, and then setting `version.Version = "foobar"`.